pkg/storage/bloom/v1/bounds.go

```diff
@@ -33,8 +64,16 @@ func (b FingerprintBounds) Hash(h hash.Hash32) error {
 	return errors.Wrap(err, "writing OwnershipRange")
 }
 
+// Addr returns the string representation of the fingerprint bounds for use in
+// content addressable storage.
+// TODO(owen-d): incorporate this into the schema so we can change it,
+// similar to `{,Parse}ExternalKey`
+func (b FingerprintBounds) Addr() string {
+	return fmt.Sprintf("%x-%x", uint64(b.Min), uint64(b.Max))
```

Comment:
This will change the string representation from a fixed length string for Min/Max to a variable length, right?

Is this what we want? When putting blocks to storage, a fixed length would make them sortable by Min fingerprint.

Reply:
IMO `fmt.Sprintf("%016x-%016x", uint64(b.Min), uint64(b.Max))` is what we want.

Reply:
oof, good point. Will fix